scipy/signal/tests/test_savitzky_golay.py

```diff
@@ -137,6 +137,19 @@ def test_sg_coeffs_deriv():
         assert_allclose(coeffs2.dot(x), d2x[pos], atol=1e-10)
 
 
+def test_sg_deriv_ge_polyorder():
+    """
+    If deriv >= polyorder, the coefficients should be all 0.
```

Comment:
Why does the docstring for the unit test claim the condition for all zero is `>=`, while the code change above uses `>`? Maybe one needs a slight adjustment?

I'm guessing you need the derivative order to be `>` so that you end up taking the derivative of a constant. Perhaps with `==` order you can end up with a non-zero constant like standard calculus?

Reply:
> Why does the docstring for the unit test claim the condition for all zero is `>=`, while the code change above uses `>`?

Because that docstring in the test is wrong. 😧   I'll fix it.

Reply:
I pushed an update with the test's docstring fixed.